makefiles/dependency_resolution.inc.mk

```diff
@@ -93,4 +93,12 @@ else
                             "don't run this on public networks!$(COLOR_RESET)" 1>&2)
     endif
   endif
+
+  # Warn about PSA Crypto
+  ifneq (,$(filter psa_crypto,$(USEMODULE)))
+    $(shell $(COLOR_ECHO) "$(COLOR_YELLOW) You are going to use the PSA Crypto module,"\
```

Comment:
Should this go to the documentation as well?

Reply:
The warnings for `stdio_udp` and `auto_init_telnet` above also fire when I use them and call `make clean`.
I have no idea how to fix this and frankly I don't care.

Reply:
Is there anywhere a TODO file or similar that says what needs to be done so we can remove this warning?
"We have a crypto API, but please don't use it for anything security critical because 👋" is a bit of an odd situation.

Reply:
@benpicco There's not really a quantifiable TODO list. The biggest issue is that I still have to add the PSA test suite (I will submit a separate PR for that).
The other problem is that I don't really trust that there are no vulnerabilities left in my code. Since apparently only one and a half people actually reviewed it, it's totally possible that there are still some hiccups left.
So the next TODO would be that another person who knows about secure coding takes a look at it.

Also I'm kind of hoping that once people start to work with this and to add their own stuff, it will become more clear, whether there's anything else that needs to be fixed.
So the idea is to leave this warning message for at least one release cycle for people to play around with it.